lib/utils/bedroomCalculator.test.ts

```diff
@@ -0,0 +1,51 @@
+import { CALCULATE_BEDROOMS } from './bedroomCalculator';
+
+const people = [
```

Comment:
Would be useful to add some more test cases here.

Reply:
Will do.